senaite-openmrs/src/test/java/com/ozonehis/eip/openmrs/senaite/processors/ServiceRequestProcessorTest.java

```diff
@@ -0,0 +1,364 @@
+/*
+ * Copyright © 2021, Ozone HIS <info@ozone-his.com>
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.ozonehis.eip.openmrs.senaite.processors;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Avoid wildcard imports. You need to configure your IDE so as not to collapse imports.